src/Azure/Azure.Quantum.Client/Authentication/IAccessTokenProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿// <copyright file="IAccessTokenProvider.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I just noticed this here; We license everything under MIT license, so we should make sure to add that to all files. Do you know why that was missing in the first place?

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

Reply:
That's probably because the Azure client came from a private repository. I'll update the headers.

Reply:
Sorry, that's my fault, I didn't add the headers when I moved these files here.

Reply:
No worries!